src/track/track.h

```diff
@@ -330,6 +330,7 @@ class Track : public QObject {
     void keyUpdated(double key);
     void keysUpdated();
     void ReplayGainUpdated(mixxx::ReplayGain replayGain);
```

Comment:
Should fix the naming of this signal in this PR?

Reply:
Done.